RecoTauTag/Configuration/python/HPSPFTaus_cff.py

```diff
@@ -80,12 +80,12 @@
 ## ByMediumIsolation
 hpsPFTauDiscriminationByMediumIsolation = hpsPFTauDiscriminationByLooseIsolation.clone()
 hpsPFTauDiscriminationByMediumIsolation.qualityCuts.isolationQualityCuts.minTrackPt = 0.8
-hpsPFTauDiscriminationByMediumIsolation.qualityCuts.isolationQualityCuts.minGammaEt = 0.8
+hpsPFTauDiscriminationByMediumIsolation.qualityCuts.isolationQualityCuts.minGammaEt = 1.0
```

Comment:
this change in threshold is applied here for all tau reconstruction: run1, phase-2, 2015-16, and the current pp of 2017.
Is this the intention or are you trying to mitigate the effects of the ECAL ZS/SR changes special for 2017 and this applies only to 2017?

Reply:
Hello,
we are aware that it is applied for tau reconstruction for all Eras; a couple words of explanation to make it more clear:
1. The change has nothing to do with ECAL ZS/SR changes - it was checked with dedicated samples that the issue observed by Tau POG with low-Pt photons is not cured by the ZS/SR fix proposed by ECAL DPG.
2. The issue is related to Run-2 data, i.e. was not observed during Run-1, but we suppose that CMSSW from 94X and newer release series will not be used to re-reconstruct Run-1 data. However, if such a re-reco is not impossible we can introduce a specialization for Run-1 - please let us know.
3. It is intended that it consistently modifies reconstruction of Run-2 data. However, we are not aware of any plan of re-reconstruction of 2015 data with the 94X+ release series. What concerns 2016 80XLegacy re-miniAOD with 94X it is in fact foreseen that the re-miniAOD will consists of re-reco of taus with 2017-like setup.
4. Taus for phase-2: it is fine that this modification is propagated there as we believe that it will help at high PU. However, tau reco for phase-2 is still worked on (for instance in the context of HGCAL and Physics TDRs) and additional changes can be expected.